packages/appstore/src/assets/svgs/trading-platform/index.tsx

```diff
@@ -1,21 +1,25 @@
 import React from 'react';
-import BinaryBot from 'Assets/svgs/trading-platform/ic-brand-binarybot.svg';
+import BinaryBot from 'Assets/svgs/trading-platform/branding/ic-branding-binarybot-dashboard.svg';
 import BinaryBotBlue from 'Assets/svgs/trading-platform/ic-appstore-binarybot-blue.svg';
-import DBot from 'Assets/svgs/trading-platform/ic-brand-dbot.svg';
+import DBot from 'Assets/svgs/trading-platform/branding/ic-branding-dbot-dashboard.svg';
 import Demo from 'Assets/svgs/trading-platform/ic-brand-demo.svg';
-import Derived from 'Assets/svgs/trading-platform/ic-appstore-derived.svg';
-import DerivGo from 'Assets/svgs/trading-platform/ic-brand-derivgo.svg';
+import Derived from 'Assets/svgs/trading-platform/branding/ic-branding-mt5-derived-dashboard.svg';
+import DerivGo from 'Assets/svgs/trading-platform/branding/ic-branding-derivgo-dashboard.svg';
 import DerivGoBlack from 'Assets/svgs/trading-platform/ic-appstore-derivgo-black.svg';
-import DerivLogo from 'Assets/svgs/trading-platform/ic-appstore-deriv-logo.svg';
+import DerivLogo from 'Assets/svgs/trading-platform/branding/ic-branding-deriv-logo.svg';
 import DerivTradingLogo from 'Assets/svgs/trading-platform/ic-appstore-deriv-trading-logo.svg';
-import DerivX from 'Assets/svgs/trading-platform/ic-appstore-derivx.svg';
+import DerivX from 'Assets/svgs/trading-platform/branding/ic-branding-derivx-dashboard.svg';
 import DropDown from 'Assets/svgs/trading-platform/drop-down.svg';
-import DTrader from 'Assets/svgs/trading-platform/ic-brand-dtrader.svg';
-import Financial from 'Assets/svgs/trading-platform/ic-appstore-financial.svg';
+import DTrader from 'Assets/svgs/trading-platform/branding/ic-branding-dtrader-dashboard.svg';
+import Financial from 'Assets/svgs/trading-platform/branding/ic-branding-mt5-financial-dashboard.svg';
 import Options from 'Assets/svgs/trading-platform/ic-appstore-options.svg';
-import SmartTrader from 'Assets/svgs/trading-platform/ic-brand-smarttrader.svg';
+import SmartTrader from 'Assets/svgs/trading-platform/branding/ic-branding-smarttrader-dashboard.svg';
 import SmartTraderBlue from 'Assets/svgs/trading-platform/ic-appstore-smarttrader-blue.svg';
 import CFDs from 'Assets/svgs/trading-platform/ic-appstore-cfds.svg';
+import DBotRebranding from 'Assets/svgs/trading-platform/ic-brand-dbot.svg';
+import DerivGoRebranding from 'Assets/svgs/trading-platform/ic-brand-derivgo.svg';
+import MT5DerivedRebranding from 'Assets/svgs/trading-platform/ic-brand-mt5-derived.svg';
+import MT5FinancialRebranding from 'Assets/svgs/trading-platform/ic-brand-mt5-financial.svg';
```

Comment:
Nit: Seems like the rest of the buttons are in alphabetical order, better to follow that